css_tasks/task11/scss/_home.scss

```diff
@@ -0,0 +1,192 @@
+.home__background {
+  position: relative;
+  :after {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    content: "";
+    background: $gradient;
+    opacity: 0.4;
+  }
+}
+.home-background__image {
+  display: block;
+  width: 100%;
+  height: 100%;
+  max-height: 100vh;
+}
+.header {
+  position: absolute;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-top: 2%;
+
+  &__logo {
+    flex-grow: 1;
+    @include whiteBoldText;
+    font-size: $titleFontSize;
+    text-decoration: none;
+  }
+}
+.header__nav {
+  flex-grow: 4;
+  ul {
+    display: flex;
+    justify-content: flex-end;
+    list-style: none;
+  }
+}
+.header-nav__item {
+  display: block;
+  margin-right: calc(11px + (42 - 11) * ((100vw - 300px) / (1920 - 300)));
+  a {
+    font-size: calc(11px + (14 - 11) * ((100vw - 300px) / (1920 - 300)));
+    font-family: $secondaryFont;
+    color: $mainBackgroundColor;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+  span {
+    vertical-align: middle;
+    cursor: pointer;
+  }
+  img {
+    width: 17px;
+    height: 15px;
+  }
+}
+
+.header-nav__item:first-child a {
+  position: relative;
+  color: $activeColor;
+  &:after {
```

Comment:
it is more hover effect, and it is possible to do without after element with just border, and not assign it to just first element, it is bad code style

Reply:
I just looked at design.

Reply:
it was more how it looks at design about some hover/active state but not fixed